app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,161 @@ function Page() {
 
                 <div><pre><code>
                   {`.grid-view {
-                    --_gap: var(--grid-list-grid-gap, 1rem);
+    --_gap: var(--grid-list-grid-gap, 1rem);
 
-                    display: grid;
+    display: grid;
 
-                    padding: var(--_gap);
-                    grid-gap: var(--_gap);
-                    grid-template-columns: repeat(4, 1fr);
-                    grid-template-rows: auto;
-                  }
-                `}
-                </code></pre>
-                </div>
+    padding: var(--_gap);
+    grid-gap: var(--_gap);
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;
+  }
+`}
+                </code></pre></div>
               </li>
             </ul>
 
-            <h2>Layout</h2>
-            <p>TBD ...</p>
+            <h2><code>Layout Container</code> (Base)</h2>
+            <p>The stanard <code>{`<LayoutContainer>`}</code> wraps content containers to provide
+            some common container within the layout container.</p>
+            <LayoutContainer style={{'--layout-container-background': 'var(--slate-200)'}}>
+              <ContentContainer>
```

Comment:
Are we planning to make our page backgrounds gray, like in the wireframes? If so, then ContentContainer should probably be given a white background. Also, I think it would be nice if we added a border-radius with a little bit of box shadow.

Reply:
I'll add those that style updates as you mention. But, on the styleguide I'm still going to deliberately override the backgrounds because it's only to demonstrate the different layout regions on the sidebar.

We should probably also decide on a style for "toolbars"

Reply:
@jupiter007 it's not really feasible to add the border and shadow, even the background color globally to `ContentContainer`. This is because of the white area in which they can be used, and the content they may contain. The primary goal for content containers is to hold content for a layout.

We can customize each content container differently with the CSS variables, on the pages where they need specific styling. ContentContainers only ensures that padding and positioning works correctly within other layouts